### PR TITLE
8294527: Some java.security.debug options missing from security docs

### PR DESCRIPTION
**Pull request by ryawalla**
Hi, small change to the java.security.debug to enhance help() with missing options.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8294527](https://bugs.openjdk.org/browse/JDK-8294527): Some java.security.debug options missing from security docs

### Reviewers
 * [Sean Mullan](https://openjdk.org/census#mullan) (@seanjmullan - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/12289/head:pull/12289` \
`$ git checkout pull/12289`

Update a local copy of the PR: \
`$ git checkout pull/12289` \
`$ git pull https://git.openjdk.org/jdk pull/12289/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 12289`

View PR using the GUI difftool: \
`$ git pr show -t 12289`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/12289.diff">https://git.openjdk.org/jdk/pull/12289.diff</a>

</details>

### CODE REVIEW

File: src/java.base/share/classes/sun/security/util/Debug.java [hunk 1]
```diff
@@ -87,10 +87,12 @@ public static void Help()
         System.err.println("pkcs11keystore");
         System.err.println("              PKCS11 KeyStore debugging");
         System.err.println("pkcs12        PKCS12 KeyStore debugging");
+        System.err.println("properties    Prints which java.security Security Properties file that the JDK uses");
```
Comment (seanjmullan) [inline src/java.base/share/classes/sun/security/util/Debug.java:90]: This prints the initial values of all security properties (whether overridden or not) and other miscellaneous useful info when parsing the java.security file. I think we can just generalize this to "Security property and configuration file debugging". The other help options are fairly general.

Comment (ryawalla) [reply src/java.base/share/classes/sun/security/util/Debug.java]: Done, thanks Sean

**Review by seanjmullan — APPROVED** [review]

**Pull request closed without merging**